documentation/examples/17-special-elements/00-svelte-self/Folder.svelte

```diff
@@ -11,7 +11,8 @@
 	}
 </script>
 
-<span class:expanded on:click={toggle}>{name}</span>
+<!-- svelte-ignore a11y-click-events-have-key-events -->
+<span class:expanded on:click={toggle} role='button' tabindex='0'>{name}</span>
```

Comment:
In this case, it would be better to make this a native `<button>` element instead of applying role + tabindex. This is because we use `role=button`, we also have to listen for key events and handle `Space` and `Enter` appropriately. It's simpler to just use the native button